ElrondNetwork/elrond-go
# Termui trie sync status

## Reasoning behind the pull request
- trie sync status missing from termui
  
## Proposed changes
- add a specific message for trie sync status(eg. `Trie sync: 12345678 node, 2.10GB state size`)

## Testing procedure
- start a node on testnet/mainnet(recommended, for a longer sync time) and check `Chain info` section on termui.. during trie sync, the above message should be displayed on second row(example below)

![Screenshot from 2022-11-01 13-17-12](https://user-images.githubusercontent.com/34831323/199224389-7354814c-903f-4daf-92fe-267b2e5c6177.png)

## Pre-requisites

Based on the [Contributing Guidelines](CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
